src/main/java/org/kaazing/netx/ws/internal/WebSocketExtensionFactory.java

```diff
@@ -48,23 +52,37 @@ public WebSocketExtensionSpi createExtension(String name, WebSocketExtensionPara
         return factory.createExtension(parameters);
```

Comment:
Do you think it would be worth passing the formatted string to let the SPI parse out the parameters instead?